Kitodo-DataManagement/src/test/resources/log4j2.xml

```diff
@@ -18,6 +18,12 @@
         </Console>
     </Appenders>
     <Loggers>
+        <Logger name="org.kitodo" level="debug" additivity="false">
+            <AppenderRef ref="STDOUT"/>
+        </Logger>
+        <Logger name="org.opensearch.node" level="info" additivity="false">
+            <AppenderRef ref="STDOUT"/>
+        </Logger>
```

Comment:
Is this really needed? Log output is already really big and CI executions got aborted if log output is to big.